Directory.Build.props

```diff
@@ -23,6 +23,15 @@
   <PropertyGroup Condition="'$(Configuration)'=='Release'">
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
+  <!-- Explicit FrameworkReference -->
+  <PropertyGroup Condition=" !($(TargetFramework.StartsWith('net5')) Or $(TargetFramework.StartsWith('net6')) Or $(TargetFramework.StartsWith('net7'))) ">
+    <ImportWindowsDesktopTargets>true</ImportWindowsDesktopTargets>
```

Comment:
So does this mean that all assemblies now have reference to use WinForms or WPF instead of just those few that needed it before?

Reply:
Good question.
I used this file for simplicity to make things compile.

I guess it will be safer to return it to the csproj.
I just don’t like the copy paste here.
What do you think is better explicit props file import or a custom property which is used automatically in Directory.Build?

Reply:
I have some idea how to solve it in a best way.
Will push soon.

Reply:
I returned explicit properties.
For .NET5 either way you need the Windows FrameworkReference since this what TFM -windows does automatically for you.

Reply:
Thank you. So is this ready to merge? Are you comfortable with your testing?

Reply:
Yes.
You can see it on your machine: http://files.rsdn.org/16901/winforms-no-tfm.zip
The net5.0 directory has the same code, however publishing makes only 18MB executable instead of 81MB of net5.0-windows.